WireAuthentication/Sources/WireAuthentication/Components/DetermineAuthMethodComponent.swift

```diff
@@ -29,6 +29,8 @@ protocol DetermineAuthMethodComponentDependency: Dependency {
     var defaultBackendEnvironment: BackendEnvironment { get }
     var defaultAPIVersion: APIVersion { get }
     var minTLSVersion: TLSVersion { get }
+    var callbackScheme: String { get }
```

Comment:
suggestion: to clarify what this is for, call it something like `ssoCallbackURLScheme`.

Reply:
changed